src/components/Inputs/WalletPassphrase.tsx

```diff
@@ -27,19 +28,20 @@ import Input from './Input'
 interface Props {
   value: string
   onChange: (passphrase: string) => void
+  className?: string
 }
 
-const passphraseLink = 'https://wiki.alephium.org/wallet/Desktop-Wallet-Guide#passphrase'
-
-const WalletPassphrase = ({ value, onChange }: Props) => (
-  <ExpandableSection sectionTitleClosed="Optional passphrase (advanced)" centered>
-    <InfoBox noMarginBottom importance="alert">
+const WalletPassphrase = ({ value, onChange, className }: Props) => (
+  <ExpandableSection sectionTitleClosed="Optional passphrase (advanced)" centered className={className}>
+    <InfoBox importance="alert">
       <p>
         <WarningEmphasis>This is an advanced feature! </WarningEmphasis>
         <br />
-        <span>Use it only if you know what you are doing. Please, consult our </span>
-        <ActionLink onClick={() => openInWebBrowser(passphraseLink)}>documentation</ActionLink>
-        <span> to learn about it.</span>
+        <span>Use it only if you know what you are doing.</span>
+        <br />
+        <span>
+          Please, consult our <Link>documentation</Link> to learn about it.
+        </span>
```

Comment:
I created the local `Link` component so that once we integrate i18n it will be easier to reason about. Lemme know what you think otherwise.